Bruin Bite.xcodeproj/project.pbxproj

```diff
@@ -704,7 +707,6 @@
 				2A03D94820B7AFA7004EFC97 /* HomeProfileStaticTableViewController.swift in Sources */,
 				770CDEB020266B6000658925 /* ActivityLevelBar.swift in Sources */,
 				BD7870A621B6793A00E46A50 /* PresentAlert.swift in Sources */,
-				BF5D97E620B97F490086D451 /* ChatPreview.swift in Sources */,
```

Comment:
Any idea why this change occurred? Did you delete a file called "ChatPreview.swift" from Xcode?

Reply:
Oh no! Let me check that out...